tests/unit/test_oci.py

```diff
@@ -585,9 +585,31 @@ def test_set_entrypoint_default(self, mock_run):
             ),
         ]
 
+    def test_set_entrypoint_default_no_verbose(self, mock_run):
```

Comment:
The two test are quite the same, why not using this : pytest.mark.parametrize

Reply:
Thanks for the suggestion! Done.